pythonforandroid/recipes/vlc/__init__.py

```diff
@@ -34,14 +34,14 @@ def prebuild_arch(self, arch):
             aar_path = join(port_dir, 'libvlc', 'build', 'outputs', 'aar')
             self.aars[arch] = aar = join(aar_path, 'libvlc-{}.aar'.format(self.version))
             warning("HINT: set path to precompiled libvlc-<ver>.aar bundle " \
-                        "in {} environment!".format(self.ENV_LIBVLC_AAR))
+                    "in {} environment!".format(self.ENV_LIBVLC_AAR))
```

Comment:
\ doesn't seem necessary here, as things are already in parentheses, but they are not harmful either.

Reply:
Good catch, I've addressed. I think there's a PEP for that one also by the way. Hopefully we can address all the other later